### DISCUSSION TITLE
Unified scheme and service delivery

### DISCUSSION THREAD

**Discussion by krishnaji-eGov** [egovernments/Digit-Core#40] [Announcements]
For the Citizen-one app until now we created the required folder structure, developed and modified the existing widgets which will match Digit and mGramseva theme and also integrated the *sqflite*(local db) and *JSON* rendering plugins, but now we need inputs exactly what we need to implement further in the first phase(which screens should be developed, which services should be integrated)

**Comment by eGovMan** [comment]
@krishnaji-eGov you should look at PGR as the first service.
